CHANGELOG.md

```diff
@@ -78,10 +86,8 @@
 ## Version 0.5.3 (2018-08-07)
 
 ### Features
-* Added `support` property to `skbio.tree.TreeNode` to store the branch support value.
-* Added `assign_supports` method to `skbio.tree.TreeNode` to extract branch support values from node labels.
-* Modified the way a node label is printed. Now it is `support:name` if both exist, or `support` or `name` if either exists.
-* Added `unpack` and `unpack_by_func` methods to `skbio.tree.TreeNode` to unpack one or multiple internal nodes. The "unpack" operation removes an internal node and regrafts its children to its parent while retaining the overall length.
+* Added `unpack` and `unpack_by_func` to `skbio.tree.TreeNode` to unpack one or multiple internal nodes. The "unpack" operation removes an internal node and regrafts its children to its parent while retaining the overall length.
```

Comment:
this looks like a duplicate of the entry above?

Reply:
But this text, and the `support` note for `TreeNode` immediately below is present both for `0.5.3` and the dev section of the changelog (see lines 9 and 17). Shouldn't these entries only be present a single time?

Reply:
Got it, I was just able to pick up on that. Should be good now.